.github/workflows/_buildpacks-release.yml

```diff
@@ -59,7 +69,7 @@ env:
 jobs:
   compile:
     name: Compile Buildpacks
-    runs-on: ubuntu-latest
+    runs-on: ${{ inputs.ip_allowlisted_runner }}
```

Comment:
The allowlisted runner is only needed for jobs that interact with the certain repositories.  The compile job does not require this.

Reply:
You need this to pull the languages-github-actions lockfile from the repo.

Reply:
It's a public repo, so not sure why that's needed? Strange :-)